source/gui/settingsDialogs.py

```diff
@@ -4613,6 +4613,14 @@ def makeSettings(self, settingsSizer):
 		self.bindHelpEvent("BrailleSpeakOnRouting", self.speakOnRoutingCheckBox)
 		self.speakOnRoutingCheckBox.Value = config.conf["braille"]["speakOnRouting"]
 
+		# Translators: The label for a setting in braille settings to speak the current line or paragraph when navigating by them with braille.
+		speakOnNavigatingText = _("Speak &line or paragraph when navigating by them with braille")
```

Comment:
IMO, the option could be reworded.
* No need to tell "word or paragraph": the feedback depends on the corresponding option
* No need to mention braille since we are already in braille panel IMO

```suggestion
		speakOnNavigatingText = _("Provide speech feedback when moving by &line")
```